Program.cs

```diff
@@ -1,18 +1,40 @@
 ﻿using System;
+using System.Collections.Generic;
 using Newtonsoft.Json;
 
 namespace ConsoleInitApp
 {
     class Program
     {
-       static bool shouldQuit = false;
-        static bool goBack = false;
+        static List<Character> Characters = new List<Character>();
+        static List<Party> Parties = new List<Party>();
+        static List<Encounter> Encounters = new List<Encounter>();
 
-        static void Main(string[] args)
+        static void Pause()
+        {
+            Console.Write("\nPress any key to continue...");
+            Console.ReadKey();
+        }
+
+        static string Prompt(string message)
         {
-            //Main menu to begin app. Simple choose option.
+            Console.WriteLine("\n{0}:", message);
+            Console.Write(">: ");
+            return Console.ReadLine();
+        }
 
-            Menu start = new Menu("Choose an option", new string[] {"Add a character", "Add a party", "Add an encounter", "Start an Encounter", "Quit" });
+        // Main menu to begin app. Simple choose option.
+        static void Main(string[] args)
+        {
+            bool shouldQuit = false;
+            Menu start = new Menu("Main menu", new string[] {
+                "Add a character",
+                "Show characters",
+                "Add a party",
+                "Add an encounter",
+                "Start an Encounter",
+                "Quit"
+            });
```

Comment:
This was just a visual formatting thing so I could keep track of what options go where.